cli/tests/testdata/compat/dyn_import_reject.out

```diff
@@ -1,2 +1,2 @@
-TypeError: Cannot load module "file:///[WILDCARD]/testdata/compat/foobar.js".
+TypeError: [ERR_MODULE_NOT_FOUND] Cannot find module '[WILDCARD]/testdata/compat/foobar.js' imported from file:///[WILDCARD]/testdata/compat/dyn_import_reject.js
```

Comment:
These changes are most likely related to how referrer is parsed in `cli/proc_state.rs`